data/map altera.txt

```diff
@@ -4357,8 +4357,8 @@ planet Elarum
 
 planet "Forta Dracone"
 	landscape land/nasa0
-	description `Considering the importance of Draconis and its Draconium deposits, the moon orbiting Draconis used as a large military base, making sure that not only the Draconis, but also the surrounding systems are kept safe from potential pirates or invaders. While the moon looks barren from space, it is actually inhabitable and a few navy owned farms can be seen while landing.`
```

Comment:
Minor style correction. Actually, the "potential" should be completely remove. You're either a pirate / invader or not.

Reply:
Then I suggest to either combine it to "potential threats" or drop the potential and just state "...from pirates and invaders".

Reply:
I think "potential threats" is the best solution, if you commit that change, I'd be happy to merge the PR.

Reply:
Done. Happy merging. :)